projects/project-1/assignment/New Short Version assignment 1

```diff
@@ -0,0 +1,199 @@
+
+-  For Bi/Bi, see Exercise 1
+
+
+<header>
+  <h3>Reflexivity/Asymmetry</h3>
+</header>
+
+- <b>A/A, Ai/Ai, B/B, Bi/Bi</b>: see Exercise 1
+
+- <b>A/Ai</b>: suppose that A is reflexive and Ai is asymmetric. By reflexivity, xAx. Since Ai is inverse A, xAix also holds.
+However, because of asymmetricity, xAix should not hold.   
+
+- <b>A/B</b>: suppose that A is reflexive and B is asymmetric. By reflexivity, xAx. However, since B is a subproperty of A meaning that all the instances of B have to be a part of
+instances of A, the same problem that occured with A/A occurs. Therefore, it is not the case that A is reflexive and B is asymmetric.
+
+- <b>A/Bi</b>: suppose that A is reflexive and Bi is asymmetric. By reflexivity, xAx. And since B is a subproperty of A xBx should hold. Since Bi is inverse B, xBix also holds.
+However, because of asymmetricity, xBix should not hold.   
+
+- <b>Ai/A</b>: suppose that Ai is reflexive and A is asymmetric. By reflexivity, xAix. Since A is inverse Ai, xAix also holds.
+However, because of asymmetricity, xAx should not hold.
+    
+- <b>Ai/B</b>: suppose that Ai is reflexive and B is asymmetric. By reflexivity, xAix. Since B is a subproperty of A that is inverse Ai, xBx should hold.
+However, because of asymmetricity, xBx should not hold.
+  
+- <b>Ai/Bi</b>: suppose that Ai is reflexive and Bi is asymmetric. The explanation for this error will be the same for A being refelxive and B being asymmetric.
+    
+- <b>B/Bi</b>: suppose that B is reflexive and Bi is asymmetric. By reflexivity, xBx. Since Bi is inverse B, xBix also holds.
+However, because of asymmetricity, if xBiy, then it is not the case that yBix.   
+
+
+Functionality / Transitivity
+
+![image](https://github.com/user-attachments/assets/94f987df-41cd-4d1c-b278-1c45ad5ba4a3)
+
+1. A/A
+If xAy and yAz (transitivity), then xAz must hold.
+However, if A is functional, x can only have one y, leading to a conflict because xAz contradicts functionality (X).
+
+3. Transitive A / Functional Ai
+If xAy implies yAx (inverse of A as Ai), and xAi​z implies zAi​y, the transitive relationship between A and Ai creates a conflict since A is functional, and x cannot relate to multiple y's (X).
+
+4. Transitive Ai / Functional A
+If xAi​y implies yAx (inverse relationship), and yAz implies xAz (transitivity), functionality is violated because x can only have one relationship, but transitivity implies multiple connections for x (X).
+
+5. Ai/Ai
+same as A/A
+
+6. Transitive B / Functional A
+If xBy (B is a subproperty of A) implies xAy, and yAz holds, then xAz should hold (transitivity).
+However, since Bi nherits functionality from A, xAz contradicts the functional nature of A, creating a conflict (X).
+
+8. Transitive B / Functional Ai
+If xBy implies xAy, and xAi​z implies zAi​y, the inverse relationship causes a conflict because B inherits transitivity and functionality from A, while Ai creates an inverse that doesn't align with these constraints (X).
```

Comment:
Why does xAiz imply xAiz?

I would simply say that xBy implies xAy, that xAiy implies yAix, that this makes Ai inverse functional, and that this creates a complex property chain